examples/native/components/ControlExamples/Boolean/Boolean.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+
+export interface Props {
+  on: boolean;
+}
+
+const styles = StyleSheet.create({
+  circle: {
+    width: 60,
+    height: 60,
+    borderRadius: 30,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  on: { backgroundColor: 'yellow' },
+  off: { backgroundColor: 'black' },
+  offText: { color: 'white' },
+});
+
+export const Switch = ({ on }: Props) => (
+  <View style={[styles.circle, on ? styles.on : styles.off]}>
+    <Text style={!on && styles.offText}>{on ? 'on' : 'off'}</Text>
```

Comment:
This behaves a bit weird for me on the Android emulator:

1. When I initially open it, everything looks fine:

![Screen Shot 2021-07-11 at 19 30 38](https://user-images.githubusercontent.com/6605505/125202899-816b9480-e27e-11eb-971e-1b993bb937c0.png)

2. If I then go to "Addons -> Controls" and toggle it off, it still renders fine:

![Screen Shot 2021-07-11 at 19 32 53](https://user-images.githubusercontent.com/6605505/125202958-cdb6d480-e27e-11eb-8de2-d218e6359e26.png)
 
3. If I again go to "Addons -> Controls" and toggle it back on, the rendering becomes wonky:

![Screen Shot 2021-07-11 at 19 31 11](https://user-images.githubusercontent.com/6605505/125202907-90524700-e27e-11eb-9707-f2f92e457620.png)

This doesn't happen on iOS. Explicitly setting the color for both off & on seems to fix this:

```
const styles = StyleSheet.create({
...
  onText: { color: 'black' },
  offText: { color: 'white' },
});
...
    <Text style={on ? styles.onText : styles.offText}>{on ? 'on' : 'off'}</Text>
```

Reply:
Oh I guess there isn't a default color on Android then. I'll change this. Thanks!